src/base/utils/fs.cpp

```diff
@@ -355,6 +355,13 @@ bool Utils::Fs::sameFileNames(const QString &first, const QString &second)
 #endif
 }
 
+bool Utils::Fs::isFileInFolder(const QString &file_path)
```

Comment:
All public class methods returns paths with '/' as separator (unless there is implementation bug).